README.md

```diff
@@ -749,6 +750,35 @@ public LockProvider lockProvider() {
 }
 ```
 
+#### Memcached (using spymemcached)
+Import
+```xml
+<dependency>
+    <groupId>net.javacrumbs.shedlock</groupId>
+    <artifactId>shedlock-provider-memcached</artifactId>
+    <version>4.32.0</version>
+</dependency>
+```
+
+and configure
+
+```java
+import net.javacrumbs.shedlock.provider.memcached.MemcachedLockProvider;
+
+...
+
+@Bean
+public LockProvider lockProvider(net.spy.memcached.MemcachedClient client) {
+    return new MemcachedLockProvider(client, ENV);
+}
+```
+
+ps:
```

Comment:
I do not understand the ps. What is it supposed to tell the user?

Reply:
We really should stress-out that due to the nature of the `memcached` there is basically no guarantee about the lock.

Reply:
P.S.